packages/functions/src/callable.test.ts

```diff
@@ -133,6 +137,94 @@ describe('Firebase Functions > Call', () => {
     stub.restore();
   });
 
+  it('app check token', async () => {
+    const authProvider = new Provider<FirebaseAuthInternalName>(
+      'auth-internal',
+      new ComponentContainer('test')
+    );
+
+    const messagingProvider = new Provider<MessagingInternalComponentName>(
+      'messaging-internal',
+      new ComponentContainer('test')
+    );
+
+    const appCheckMock: FirebaseAppCheckInternal = {
+      getToken: async () => ({ token: 'app-check-token' })
+    } as unknown as FirebaseAppCheckInternal;
+    const appCheckProvider = new Provider<AppCheckInternalComponentName>(
+      'app-check-internal',
+      new ComponentContainer('test')
+    );
+    appCheckProvider.setComponent(
+      new Component(
+        'app-check-internal',
+        () => appCheckMock,
+        ComponentType.PRIVATE
+      )
+    );
+    const functions = createTestService(
+      app,
+      region,
+      authProvider,
```

Comment:
I don't think you have to define authProvider or messagingProvider, `createTestService` has defaults for all 3 of these, you can just put `undefined, undefined, appCheckProvider`. That seems to be what the test below does when it only cares about messaging.